enrolment-ui/src/i18n.js

```diff
@@ -1,25 +1,26 @@
 import i18n from 'i18next';
-import Backend from 'i18next-xhr-backend';
-import { reactI18nextModule } from 'react-i18next';
+import HttpApi from 'i18next-http-backend';
+import { initReactI18next } from 'react-i18next';
+
+// TODO: make sure multilingual still works
 
 i18n
-  .use(Backend)
-  .use(reactI18nextModule)
+  .use(HttpApi)
+  .use(initReactI18next)
   .init({
     fallbackLng: 'en',
-
-    // have a common namespace used around the full app
-    ns: ['translations'],
```

Comment:
Removing this to avoid unnecessary additional complexity.
The package takes 'translation' as default, hence renaming the JSON files.